crates/bevy_scene/src/dynamic_scene_builder.rs

```diff
@@ -31,7 +31,7 @@ use std::collections::BTreeMap;
 /// let dynamic_scene = builder.build();
 /// ```
 pub struct DynamicSceneBuilder<'w> {
-    extracted_scene: BTreeMap<u32, DynamicEntity>,
+    extracted_scene: BTreeMap<u64, DynamicEntity>,
```

Comment:
Is it more performant to convert the `Entity` to `u64` rather than just using `Entity` as the key?

Reply:
Nope, this was just missed when shuffling between `Entity` and `u64` across different PR revisions.